platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/problems/ProblemsSummariesEvent.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.tooling.events.problems;
+
+import org.gradle.api.Incubating;
+
+/**
+ * The Event capturing all problems summaries
```

Comment:
❌ Missing javadoc:
- explain when the client can expect this event.
- and how the content should be interpreted.

Reply:
This is still relevant.